editoast/search.yml

```diff
@@ -20,3 +20,29 @@ operationalpoint:
       ch: opm.data#>>'{extensions,sncf,ch}'
       geographic: ST_AsGeoJSON(ST_Transform(opl.geographic, 4326))::json
       schematic: ST_AsGeoJSON(ST_Transform(opl.schematic, 4326))::json
+
+track:
+  table: osrd_search_track
+  columns:
+    infra_id: integer
+    line_name: string
+    line_code: integer
+  result:
+    columns:
+      infra_id: osrd_search_track.infra_id
+      line_code: osrd_search_track.line_code
+      line_name: osrd_search_track.unprocessed_line_name
+
+
+geo_track:
+  table: osrd_search_track
+  columns:
+    infra_id: integer
+    line_code: integer
+  result:
+    columns:
+      infra_id: osrd_search_track.infra_id
+      line_code: osrd_search_track.line_code
+      line_name: osrd_search_track.unprocessed_line_name
+      geographic: to_jsonb(ST_Collect(osrd_search_track.geo_sections))
+      schematic: to_jsonb(ST_Collect(osrd_search_track.sch_sections))
```

Comment:
Is it fast enough to ST_Collect at the last moment?

Reply:
yes